### PR TITLE
Fix #408 Draft for a getting started page

### LINKED ISSUE #408: Add Readme for absolute beginners
Since we are an open-source project, people who are completely new to programming can also contribute. It would be good to have a Readme for absolute beginners on how to get started (with links to installing Git, VSCode, etc) that can be used by those interested, but also at hacks and so forth, as instructions.

### PR DESCRIPTION
**Pull request by LudwikJaniuk**

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+There are many ways to use git and everyone has their favorites. One suggestion is to use [GitHub Desktop](https://docs.github.com/en/desktop) which is a quick and simple way to get started. The following guide shows you how to "clone" a "repository" with GitHub Desktop. The term "clone" means to download a copy that is managed by git, and a "repository" is what we call that copy, both locally and on GitHub.
+
+ * https://docs.github.com/en/desktop/adding-and-cloning-repositories/cloning-a-repository-from-github-to-github-desktop
+
+No matter if you use the Desktop app, ZIP, or some other method, you need to have a copy of the code somewhere on your computer in the end. The rest of the instructions assume you have located that directory and moved inside it using a terminal. 
+
+## Installing other dependencies
+
+To build and run klimatkollen, some dependencies are required apart from Node.js itself. The main ones are as follows:
```
Comment (elvbom) [inline doc/getting-started.md:36]: Klimatkollen in title case

**Review by elvbom — COMMENTED** [review]
Great work!

File: README.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ Please climb onboard to help us set our climate data free! #Klimatkollen /#FreeC
 
 ## Get started
 
+Detailed setup instructions are available in [doc/getting-started.md](doc/getting-started.md), including how to install the dependencies.
```
Comment (elvbom) [inline README.md:19]: Perhaps an even clearer wording regarding the fact that these are instructions for absolute beginners?

Comment (elvbom) [reply README.md:19]: "For absolute beginners, step-by-step setup instructions, including how to install all necessary dependencies, can be found in [doc/getting-started.md]."
?

File: doc/getting-started.md (hunk 1 above)
Comment (LudwikJaniuk) [reply doc/getting-started.md]: Resolved

File: README.md (hunk 2 above)
Comment (LudwikJaniuk) [reply README.md]: I adopted some of your wording but feel averse to the "absolute beginners" label as it's subjective and not very specific.

**Review by elvbom — APPROVED** [review]

**Pull request merged**